assets/javascripts/initializers/discourse-post-event-decorator.js.es6

```diff
@@ -195,7 +195,7 @@ function initializeDiscoursePostEventDecorator(api) {
       if (helper) {
         const post = helper.getModel();
 
-        if (post.event) {
+        if (post?.event) {
```

Comment:
Wow, that's a thing in ES6?

Reply:
sir yes sir, it's part of my work on updating babel